sound/pci/hda/hda_intel.c

```diff
@@ -2116,9 +2116,10 @@ static int azx_probe(struct pci_dev *pci,
 	 */
 	if (dmic_detect) {
 		err = snd_intel_dsp_driver_probe(pci);
-		if (err != SND_INTEL_DSP_DRIVER_ANY &&
-		    err != SND_INTEL_DSP_DRIVER_LEGACY)
+		if (err != SND_INTEL_DSP_DRIVER_ANY && err != SND_INTEL_DSP_DRIVER_LEGACY) {
+			dev_dbg(&pci->dev, "Hdaudio driver not selected, aborting probe\n");
```

Comment:
I find this already confusing enough:
```
linux$ grep -r HDAudio sound/ include/sound/ Documentation/sound/ | wc -l
25
linux$ grep -r HDaudio sound/ include/sound/ Documentation/sound/ | wc -l
21
```
and no "Hdaudio" hits *yet*. Please, let's not add one more capitalisation...

Reply:
Good catch, the comments should always be HDAudio, will fix this PR and fix all this.